docs/website/docs/plus/features/project/overview.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Project Overview
+description: Generate dlt projects in yaml
+---
+
+# Project
+
+
+<img src="https://storage.googleapis.com/dlt-blog-images/plus/dlt_plus_projects.png" width="500"/>
+
+[dlt+ Project](../../core-concepts/project.md) provides a structured and opinionated approach to organizing data workflows while implementing best practices for data engineering teams. dlt+ Project automates key processes such as data loading, data transformations, data catalogs, and data governance, and enables different members of the data teams to work more easily with each other.
+
+With dlt+ Project, you can efficiently manage your data workflows by:
+
+1. [Using a declarative `dlt.yml` file](#the-dlt-manifest-file-dltyml) to define sources, destinations, pipelines, and transformations.
+2. Configuring [different profiles](../../core-concepts/profiles.md) for various use cases and environments.
+3. Ensuring data quality by defining tests with [dlt+ tests utils](../quality/tests.md).
+4. Packaging your project as a Python package and distributing it via PyPI or a git repository [Coming Soon!]
+
+This structured approach allows teams to work efficiently while maintaining flexibility and control over their data workflows.
+
+## Project structure
+
+A dlt+ Project has the following general structure:
+```text
+├── .dlt/                 # folder containing dlt configurations and profile settings
+│   ├── config.toml
+│   ├── dev.secrets.toml  # credentials for access profile 'dev'
+│   └── secrets.toml
+├── _data/                # local storage for your project, excluded from git
+├── sources/              # modules containing the source code for sources
+│   └── github.py         # source code for a GitHub source
+├── transformations/      # modules containing the source code for transformations
+├── .gitignore
+└── dlt.yml               # the main project manifest
+```
+
+## The dlt manifest file (dlt.yml)
+
+The main component of a dlt+ Project is the dlt manifest file (`dlt.yml`). It marks the root of your project and contains the main configurations. Here you can declare all of your data platform entities in a YAML format. It contains the following sections:
+
+### Sources
+
+This section lets you define [sources](../project/source-configuration.md)  either declaratively or by referencing an implementation from a Python module inside `sources/`. In the example below, two sources are declared:
```

Comment:
I suggest to state more clearly that more details about sources configuration can be found at the `../project/source-configuration.md`